franklin/builder/admin.py

```diff
@@ -1,10 +1,23 @@
 from django.contrib import admin
 
-from .models import BranchBuild, Environment, Owner, Site, TagBuild
+from .models import BranchBuild, Deploy, Environment, Owner, Site, TagBuild
 
 
-admin.site.register(BranchBuild)
-admin.site.register(Environment)
+class EnvironmentBuildInline(admin.TabularInline):
+    model = Deploy
+    extra = 1
+
+
+class BranchBuildAdmin(admin.ModelAdmin):
+    inlines = (EnvironmentBuildInline, )
+
+
+class EnvironmentAdmin(admin.ModelAdmin):
+    inlines = (EnvironmentBuildInline, )
```

Comment:
This is a point of interest:
I've added a "through" model to manage the ManyToMany relationship between the Environment and Build models. (it stores the `deployed` datetime so we can have history of deployment events) Here we add it to admin as a `TabularInline` so we can see this relationship when viewing either a build or an environment from the admin.